versionutils/repo.go

```diff
@@ -155,3 +157,19 @@ func ParseFullTomlFromDir(relativeDir string) (*TomlWrapper, error) {
 func ParseFullToml() (*TomlWrapper, error) {
 	return ParseFullTomlFromDir("")
 }
+
+func getVersionFromTree(tomlTree *toml.Tree, pkgName string) (found bool, version string) {
```

Comment:
feels more idiomatic to reverse the order of the return args

Reply:
e.g. checking if a key is valid in a golang map:
```golang
if val, ok := map[key]; ok {
  ...
}
```

Reply:
Right! Will update